p2p/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name       = "p2p"
+version    = "0.1.0"
+edition    = "2018"
+license    = "Apache-2.0"
+repository = "https://github.com/informalsystems/tendermint-rs"
+readme     = "README.md"
+keywords   = ["p2p", "tendermint", "cosmos"]
+authors    = [
+  "Tony Arcieri <tony@iqlusion.io>",
+  "Ismail Khoffi <Ismail.Khoffi@gmail.com>"
+]
+
+description = """
+    The Tendermint P2P stack.
+    """
+
+[dependencies]
+chacha20poly1305 = "0.7"
+ed25519-dalek = "1"
+eyre = "0.6"
+hkdf = "0.10.0"
+merlin = "2"
+prost = "0.6"
+prost-amino = "0.6"
+prost-amino-derive = "0.6"
```

Comment:
Not necessarily required for this PR, but you might consider adding an `amino` cargo feature and making these dependencies `optional`

Reply:
Should we not be able to remove these altogether?

Reply:
Right, are you trying to maintain a backwards compatible KMS, so one version can run on both pre and post stargate? I actually don't see how that would be possible since tendermint-rs isn't backwards compatible? I too would probably prefer to drop all the legacy support if possible.

Reply:
I actually removed legacy versions in the first commit, but decided to bring them back in https://github.com/informalsystems/tendermint-rs/pull/696/commits/24b16ba928a145dfbf3d15ec8c167ee21d3ce420 exactly because TMKMS breakage.

Reply:
@ebuchman TMKMS now vendors the old privval Amino message types in addition to using `tendermint`/ `tendermint-proto` v0.17.0-rc3 and is able to sign on all Tendermint versions.

Reply:
Wow nice work!